src/renderers/WebGLRenderer.js

```diff
@@ -1430,7 +1430,8 @@ class WebGLRenderer {
 					samples: 4,
 					stencilBuffer: stencil,
 					resolveDepthBuffer: false,
-					resolveStencilBuffer: false
+					resolveStencilBuffer: false,
+					colorSpace: ColorManagement.workingColorSpace,
```

Comment:
Hmm, it hadn't occurred to me that the default color space of a render target is `THREE.NoColorSpace`! Good catch, and I expect that there my be some other places we may need to fix similar issues.